conjure-java-undertow-runtime/src/main/java/com/palantir/conjure/java/undertow/runtime/Encodings.java

```diff
@@ -102,7 +103,7 @@ public boolean supportsContentType(String contentType) {
                 // TODO(ckozak): support wildcards? See javax.ws.rs.core.MediaType.isCompatible
                 return contentType != null
                         // Use startsWith to avoid failures due to charset
-                        && contentType.startsWith(CONTENT_TYPE);
+                        && contentType.toLowerCase(Locale.ROOT).startsWith(CONTENT_TYPE);
```

Comment:
We may want to break out a utility function to loop over characters using `Characters.toLowerCase` and avoid potential re-allocation of the content-type string.